packages/core-mobile/app/screens/send/hooks/usePVMSend.ts

```diff
@@ -70,19 +128,43 @@ const usePVMSend: SendAdapterPVM = ({
   )
 
   const validate = useCallback(async () => {
+    if (!provider) return
+    setError(undefined)
+
     try {
+      const unsignedTx = await createSendPTx(maxFee)
+      const tx = await Avalanche.parseAvalancheTx(
+        unsignedTx,
+        provider,
+        fromAddress
+      )
+      setEstimatedFee(tx.txFee)
```

Comment:
this should live outside (like in another useState/useEffect combo) the validate logic so that we don't pollute the validate logic (this pr https://github.com/ava-labs/core-mobile/pull/2090 already cleaned it up). also, that way, we can go ahead and calculate the fee early.